rootfs/templates/wrapper

```diff
@@ -197,18 +197,18 @@ function use() {
 
```

Comment:
Do you also need to update `rootfs/sbin/docker` ?

https://github.com/cloudposse/geodesic/blob/6bd1a5125cd32e9bcf71db0d10bda053dfbc4d69/rootfs/sbin/docker#L10

Reply:
or create a `rootfs/sbin/podman` ?

Reply:
Probably... I'm assuming that is for DinD? That's likely low on my hit list for this PR, but is a good one to track for a future fix.